### PR TITLE
Fix cursor example scenes

### PR DESCRIPTION
**Pull request by noalak**
Use the cursor matching the scene name rather than DefaultCursor
Sort scene objects identically

### CODE REVIEW

File: ProjectSettings/ProjectVersion.txt [hunk 1]
```diff
@@ -1 +1 @@
-m_EditorVersion: 2017.2.0f3-MRTP
+m_EditorVersion: 2017.2.0f3-MRTP3
```
Comment (DDReaper) [inline ProjectSettings/ProjectVersion.txt]: @StephenHodgson Not sure about updating the Project version again.  Perhaps we should also remove this file from the  repo.

In any case, this PR shouldn't include changing this file.

**Comment by noalak** [comment]
Project version excluded.

File: Assets/HoloToolkit-Examples/Input/Scenes/BasicCursor.unity [hunk 2]
```diff
@@ -308,7 +304,7 @@ Prefab:
         type: 2}
       propertyPath: Cursor
       value: 
-      objectReference: {fileID: 531712469}
+      objectReference: {fileID: 0}
```
Comment (keveleigh) [inline Assets/HoloToolkit-Examples/Input/Scenes/BasicCursor.unity]: This reference should be replaced with the new cursor on Managers -> InputManager's SimpleSinglePointerSelector script.

File: Assets/HoloToolkit-Examples/Input/Scenes/Cursor.unity [hunk 3]
```diff
@@ -390,7 +388,7 @@ Prefab:
         type: 2}
       propertyPath: Cursor
       value: 
-      objectReference: {fileID: 224284989}
+      objectReference: {fileID: 0}
```
Comment (keveleigh) [inline Assets/HoloToolkit-Examples/Input/Scenes/Cursor.unity]: This reference should be replaced with the new cursor on Managers -> InputManager's SimpleSinglePointerSelector script.

**Comment by noalak** [comment]
Done!
Didn't notice it when testing due to automatic search of cursor, good catch!

**Pull request merged**